src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -169,36 +187,79 @@ public void apply(VariantContext vc, ReadsContext reads, ReferenceContext ref, F
     private VariantContext regenotypeVC(final VariantContext originalVC, final ReferenceContext ref, final FeatureContext features, boolean includeNonVariants) {
         Utils.nonNull(originalVC);
 
-        VariantContext result = originalVC;
```

Comment:
Can you confirm that all cases in the previous code that would have ultimately resulted in `result == originalVC` are preserved?